KarhooUISDK.xcodeproj/xcshareddata/xcschemes/KarhooUISDK.xcscheme

```diff
@@ -21,7 +21,7 @@
             </BuildableReference>
          </BuildActionEntry>
          <BuildActionEntry
-            buildForTesting = "NO"
+            buildForTesting = "YES"
```

Comment:
build for testing?

Reply:
It's related to the Client app, should be safe to merge with no consequences to the DP integrations.